ui/app/AppLayouts/Onboarding2/stores/OnboardingStore.qml

```diff
@@ -37,6 +38,10 @@ QtObject {
         return d.onboardingModuleInst.setPin(pin)
     }
 
+    function setPuk(puk: string) { // -> bool
+        return d.onboardingModuleInst.setPuk(puk)
```

Comment:
FYI, I'm changing a lot of those functions in my own branch because the keycard functions are better used async since they can take a second and they would freeze the UI.

Just letting you know for the future or if you want to change it already for this one.

Reply:
OK then we'll have to introduce all sorts of "loading" states to the UI, whenever we're using the `setPin()` to validate the user entry